js/authentication.md

```diff
@@ -561,6 +561,115 @@ export default class App extends React.Component {
 }
 ```
 
+#### Federated with Auth0
+
+You can use `Auth0` as one of the provider of your Cognito Federated Identity Pool so that you can allow your Auth0 users to have access to your AWS resources. 
+
+Step 1. Learn [how to integrate Auth0 with Cognito Federated Identity Pool](https://auth0.com/docs/integrations/integrating-auth0-amazon-cognito-mobile-apps)
+
+Step 2. Login with `Auth0` and then use the id token to get the AWS credentials from `Cognito Federated Idneitty Pool` by `Auth.federatedSignIn`, for example:
```

Comment:
Login with `Auth0`, then use the id token returned to get AWS credentials from `Cognito Federated Identity Pools` using the `Auth.federatedSignIn` method:

Reply:
Done